.github/workflows/tests.yml

```diff
@@ -60,7 +60,11 @@ jobs:
         run: pip install git+https://github.com/h5py/h5py
 
       - name: Install versioned-hdf5
-        run: pip install .
+        run: |
+          pip install -v cython meson-python setuptools-scm
+
+          # Build without isolation, so that we target the h5py version we just installed
+          pip install -v . --no-build-isolation
```

Comment:
I don't get the comment. What does "target" a version mean? How does this differ from using build isolation?

Reply:
Leftover from some debugging.